src/main/java/at/hannibal2/skyhanni/features/event/chocolatefactory/ChocolateFactoryShortcut.kt

```diff
@@ -36,7 +36,7 @@ class ChocolateFactoryShortcut {
 
     @SubscribeEvent
     fun onInventoryOpen(event: InventoryFullyOpenedEvent) {
-        if (RiftAPI.inRift()) return
+        if (LorenzUtils.inAnyIsland(IslandType.THE_RIFT, IslandType.KUUDRA_ARENA, IslandType.CATACOMBS, IslandType.MINESHAFT)) return
         if (!LorenzUtils.inSkyBlock) return
```

Comment:
The not in SkyBlock precondition should happen before the Skyblock Island specific precondition; I know it was inverted before you started editing but while you are here, could you flip it?

Reply:
Better yet. InAntIsland checks if you are in skyblock already do that line can be completely removed

Reply:
Done.